lua/mason-lspconfig/mappings/filetype.lua

```diff
@@ -238,12 +238,13 @@ return {
   xsd = { "lemminx" },
   xsl = { "lemminx" },
   xslt = { "lemminx" },
-  yaml = { "azure_pipelines_ls", "hydra_lsp", "snyk_ls", "spectral", "yamlls" },
+  yaml = { "azure_pipelines_ls", "github_actions_ls", "hydra_lsp", "snyk_ls", "spectral", "yamlls" },
```

Comment:
I think `yaml.github` should suffice, we don't need to register it for all YAML files. then we can even get rid of the runtime check in lspconfig `root_dir`

Reply:
I removed it from `yaml` see: 584b05aeaf610d81349b84c766c45956a329d029

> I guess this would need to be registered somewhere, not sure where it's done

Yeah I wonder too, it would be great if users wouldn't have to worry about it. Should there be some init script for like `/lua/mason-lspconfig/server_configurations/pylsp/init.lua`?

Reply:
I looked it up for the existing GitLab CI LS and there it is also the user's responsibility to add the filetype pattern. I guess if it gains enough traction it might be worth upstreaming into Neovim core.
For now I am just using my own little wrapper plugin https://github.com/disrupted/github-actions.nvim which defines and lazy-loads using ftdetect

Reply:
Resolving this, it will be documented as part of the config: https://github.com/neovim/nvim-lspconfig/pull/3551